rust/ballista/ui/scheduler/README.md

```diff
@@ -0,0 +1,26 @@
+# Ballista UI
+
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```

Comment:
I'm curious ... why Yarn? 

If we want to support yarn then we should add some documentation here such as "What is Yarn" and "How to install Yarn" but I am concerned that adding Yarn here implies we need to support Ballista on Yarn as well.

What would the alternatives be here for local testing? Cargo run? Docker compose?

Reply:
Hi @andygrove , [Yarn](https://yarnpkg.com/) is a package manager for node js similar (or alternative) to npm. I plan to add docker compose files, but for local development using yarn or npm is better because they provide hot reloading.

Reply:
Thanks, and what is the benefit of yarn over npm here? Sorry for the basic questions but I do not have experience in this ecosystem.

Reply:
I'm only familiar with yarn in the context of running Spark clusters in yarn.

Reply:
I have seen people having the same confusion between Hadoop yarn & node js yarn. I will add the link to yarn website to clear because most people here are from Hadoop background. Yarn is much faster than npm, does deduplication well, pinned resolutions (similar to Cargo.lock, it has yarn.lock) etc. Though npm is catching up these days.

Reply:
Done. Added a link to yarnpkg.com